presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -429,6 +433,54 @@ public void testSplitsGenerationWithAbortedTransactions()
         }
     }
 
+    @Test
+    public void testFullAcidTableWithOriginalFilesFails()
+            throws Exception
+    {
+        java.nio.file.Path tablePath = Files.createTempDirectory(UUID.randomUUID().toString());
+        Table table = table(
+                tablePath.toString(),
+                ImmutableList.of(),
+                Optional.empty(),
+                ImmutableMap.of("transactional", "true"));
+
+        List<String> filePaths = ImmutableList.of(
+                tablePath + "/000000_1",
+                tablePath + "/delta_0000002_0000002_0000/bucket_00000");
+
+        try {
+            for (String path : filePaths) {
+                File file = new File(path);
+                file.getParentFile().mkdirs();
```

Comment:
my intellij reports a warning `Java | Probable bugs | Result of method call ignored`.
I know a test would fail if this fails, but please `checkState` so that there is no warning.